AutomatedTesting/Gem/PythonTests/scripting/ScriptEvents_AllParamDatatypes_CreationSuccess.py

```diff
@@ -44,7 +44,7 @@ def TestScriptEvents_AllParamDatatypes_CreationSuccess():
     qtpy_asset_editor = qtpy_o3de_editor.open_asset_editor()
 
     # 2) Create new Script Event Asset
-    qtpy_asset_editor.click_menu_bar_option("Script Events")
+    qtpy_asset_editor.click_menu_bar_option(SCRIPT_EVENT_UI)
```

Comment:
`SCRIPT_CANVAS_UI` translates to `"Script Canvas"`

Shouldn't this be a const that references `"Script Events"`?

Reply:
switched this to the correct constant